intellij/src/saros/intellij/ui/menu/ShareWithUserAction.java

```diff
@@ -61,21 +63,21 @@ public void actionPerformed(AnActionEvent e) {
 
     // We allow only completely shared projects, so no need to check
     // for partially shared ones.
-    List<IResource> resources = Arrays.asList(getModuleFromVirtFile(virtualFile, e.getProject()));
+    List<IResource> resources = Arrays.asList(getModuleForVirtualFile(virtualFile, e.getProject()));
 
     List<JID> contacts = Arrays.asList(userJID);
 
     SharedIDEContext.preregisterProject(project);
     CollaborationUtils.startSession(resources, contacts);
   }
 
-  private IResource getModuleFromVirtFile(VirtualFile virtFile, Project project) {
+  private IResource getModuleForVirtualFile(VirtualFile virtualFile, Project project) {
+    ProjectFileIndex projectFileIndex = ProjectFileIndex.getInstance(project);
 
-    Module module = ProjectFileIndex.SERVICE.getInstance(project).getModuleForFile(virtFile);
```

Comment:
Is there a difference between `ProjectFileIndex.SERVICE.getInstance(project)` and `ProjectFileIndex.getInstance(project)`? You old version used the `SERVICE` version.

Reply:
With the current implementation, it makes no difference. `ProjectFileIndex.SERVICE.getInstance(project)` internally just calls `ProjectFileIndex.getInstance(project)` (see [source](https://github.com/JetBrains/intellij-community/blob/master/platform/projectModel-api/src/com/intellij/openapi/roots/ProjectFileIndex.java#L25)).

Maybe this mattered at some point in the past? The usage was added years ago and the IntelliJ API changed a lot since then.